release-schema.json

```
@@ -4,36 +4,77 @@
-              "description": "The estimated date of publication of the next sequential notice.",
-              "type": [
-                "string",
-                "null"
-              ],
-              "format": "date-time"
-            },
-            "documentAvailabilityPeriod": {
-              "title": "Document availability period",
-              "description": "The period during which documents can be accessed, or during which requests for access to documents can be submitted.",
-              "$ref": "#/definitions/Period"
-            }
-          },
-          "minProperties": 1
+          "description": "Modalities of communication about key events for the tender",
+          "$ref": "#/definitions/Communication"
         }
       }
+    },
+    "Lot": {
+      "properties": {
+        "communication": {
+          "title": "Communication",
+          "description": "Modalities of communication about key events for the lot",
+          "$ref": "#/definitions/Communication"
+        }
+      }
+    },
+    "Communication": {
+      "title": "Communication",
+      "description": "Modalities of communication about key events.",
+      "type": "object",
+      "properties": {
+        "atypicalToolName": {
+          "title": "Atypical tool name",
+          "description": "Electronic communication with the procuring entity requires the use of a tool or device that is not generally available. This is the name of this tool or device.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "minLength": 1
+        },
+        "atypicalToolUrl": {
+          "title": "Atypical tool URL",
+          "description": "Electronic communication with the procuring entity requires the use of tools and devices that are not generally available. These tools are accessible via this URL.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "uri"
+        },
+        "noticeRequestedPublicationDate": {
+          "title": "Notice preferred publication date",
+          "description": "The preferred date of publication of the notice (e.g. to avoid publication during a national holiday)",
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "date-time"
+        },
+        "futureNoticeDate": {
+          "title": "Future notice date",
+          "description": "The estimated date of publication of the next sequential notice. For a planning notice this will be the competition notice.",
+          "type": [
+            "string",
+            "null"
+          ],
+          "format": "date-time"
+        },
+        "invitationToConfirmInterest": {
+          "title": "Invitation to confirm interest",
+          "description": "The estimated period of dispatch of the invitations to confirm interest in participation. This field is applicable only when the release is a planning call for competition notice.",
```

Based on https://eur-lex.europa.eu/legal-content/EN/ALL/?uri=CELEX%3A32014L0024

> Where a prior information notice is used as a call for competition pursuant to Article 48(2), contracting authorities shall simultaneously and in writing invite the economic operators which have expressed their interest to confirm their continuing interest.

```suggestion
          "description": "The estimated date of dispatch from the buyer to potential suppliers of the invitations to confirm interest. For example, when a planning notice is used as a call for competition, the buyer might simultaneously invite the potential suppliers that have expressed their interest to confirm their continuing interest.",
```